app/services/ems_infra_dashboard_service.rb

```diff
@@ -125,12 +125,13 @@ def ems_utilization
   end
 
   def daily_provider_metrics
-    current_user = @controller.current_user
-    tp = TimeProfile.profile_for_user_tz(current_user.id, current_user.get_timezone) || TimeProfile.default_time_profile
-
-    @daily_metrics ||= Metric::Helper.find_for_interval_name('daily', tp)
-                                     .where(:resource => (@ems || ManageIQ::Providers::InfraManager.all))
-                                     .where('timestamp > ?', 30.days.ago.utc).order('timestamp')
+    @daily_provider_metrics ||= begin
```

Comment:
I think I deleted this comment by mistake before. This variable used to be `@daily_metrics`. Does it matter that it changed to `@daily_provider_metrics` here?